Libraries/acdriver/Sources/Compile/Output.cpp

```diff
@@ -29,12 +29,14 @@ Output(
     std::string const &root,
     Format format,
     ext::optional<std::string> const &appIcon,
-    ext::optional<std::string> const &launchImage) :
-    _root          (root),
-    _format        (format),
-    _appIcon       (appIcon),
-    _launchImage   (launchImage),
-    _additionalInfo(plist::Dictionary::New())
+    ext::optional<std::string> const &launchImage,
+    ExtraImageTypeSet allowedExtraImageTypes) :
```

Comment:
ExtraImageTypeSet is a nontrivial object -- pass by reference?

Reply:
Thanks :D